source/beamline/SXRMB/SXRMBBrukerDetector.cpp

```diff
@@ -28,6 +28,7 @@ SXRMBBrukerDetector::SXRMBBrukerDetector(const QString &name, const QString &des
 
 	// Stuff specialized by SXRMBBrukerDetector
 	deadTimeControl_ = new AMReadOnlyPVControl("Dead Time", "mca1606-B10-03:mca1.DTIM", this);
+	allControls_->addControl(deadTimeControl_);
 	connect(deadTimeControl_, SIGNAL(connected(bool)), this, SLOT(onDeadTimeControlConnected(bool)));
```

Comment:
@dretrex in the slot, we emit the deadtimeChanged() signal, which will trigger the UI to update the value.
Adding to the control set will only update the connectivity status of the bruker detector